legacy/builder/create_cmake_rule.go

```diff
@@ -125,22 +127,36 @@ func (s *ExportProjectCMake) Run(ctx *types.Context) error {
 		fmt.Println(err)
 	}
 
-	// Use old ctags method to generate export file
-	commands := []types.Command{
-		//&ContainerMergeCopySketchFiles{},
-		&ContainerAddPrototypes{},
```

Comment:
`ContainerAddPrototypes` is still required, I think?

Reply:
I think it's not necessary because the only goal of `create_cmake_rule.go` is to generate the cmake and all the folder/files structure in order to make the compiling process possible. So the sketch I'm using it's already preprocessed and it's only copied from the sketch folder.

Reply:
are you sure? have you tried to build something like:

```
void setup() {
   forwardFunction();
}
void loop() {
}
void forwardFunction() {
}
```

If it builds OK, then we can go ahead and merge.

Reply:
Yeah, it's not working with the code you provided:

Reply:
I see the prototype of `forwardFunction`, what's the error exactly?

Reply:
There is no error, I probably misunderstood the expected behaviour

Reply:
ok then, if the sketch compiles, the PR looks good to me.